src/App.tsx

```diff
@@ -36,14 +36,36 @@ const DEFAULT_TODO_LIST: TodoList = {
 function App() {
   const [todoListItems, setTodoListItems] = useState<TodoItemType[]>(DEFAULT_TODO_LIST.todos);
 
-  const handleTodoItemChecked = (index: number, newCheckedState: boolean) => {
+  function handleTodoItemChecked(index: number, newCheckedState: boolean) {
     const todoListItem = todoListItems[index];
 
     todoListItem.isDone = newCheckedState;
 
     todoListItems[index] = todoListItem;
 
     setTodoListItems([...todoListItems]);
+
+  }
+
+  function handleTodoItemDelete(index: number) {
+    todoListItems.splice(index, 1);
+    setTodoListItems([...todoListItems]);
+  }
+
+  function handleTodoItemAdd() {
```

Comment:
There's no semi-colons at the end of lines 56, 67, and 68 on these lines. This is a nitpick, but they're generally good for keeping track of when a block of code ends and a new one starts!

Please add :)

Reply:
handled in [77230de](https://github.com/ZachWitzig/todoist/pull/3/commits/77230de0aa48039c04a2d2707327e22d807c5848)